src/ui/qgsdatadefinedsizelegendwidget.ui

```diff
@@ -218,6 +218,7 @@
  </tabstops>
  <resources>
   <include location="../../images/images.qrc"/>
+  <include location="../../images/images.qrc"/>
```

Comment:
Is this needed or is it a typo?

Reply:
Hmm, I edited it using Design Mode in Qt Creator. Perhaps it's a bug from the Qt Creator? I will delete the duplicate manually then.

I checked just now, there is a lot of this kind of issue in the other files (e.g. https://github.com/qgis/QGIS/blob/master/src/ui/qgsstatisticalsummarybase.ui#L174)

Reply:
> Hmm, I edited it using Design Mode in Qt Creator. Perhaps it's a bug from the Qt Creator? I will delete the duplicate manually then.
> 
> I checked just now, there is a lot of this kind of issue in the other files (e.g. https://github.com/qgis/QGIS/blob/master/src/ui/qgsstatisticalsummarybase.ui#L174)

You are right. It seems to me there are 17 .ui files with more than one `  <include location="../../images/images.qrc"/>` lines.

Reply:
>Perhaps it's a bug from the Qt Creator?

Yeah it is -- it randomly just duplicates these lines for some reason

Reply:
(they are harmless, but annoying)

Reply:
https://github.com/qgis/QGIS/pull/47170